railties/test/generators/app_generator_test.rb

```diff
@@ -728,15 +728,9 @@ def test_skip_active_job_option
   def test_skip_javascript_option
     generator([destination_root], skip_javascript: true)
 
-    command_check = -> command, *_ do
-      if command == "importmap:install"
-        flunk "`importmap:install` expected to not be called."
-      end
-    end
+    assert_nil @rails_commands, "`importmap:install` expected to not be called."
 
-    generator.stub(:rails_command, command_check) do
-      run_generator_instance
-    end
+    run_generator_instance
```

Comment:
Here, you are asserting before running the generator, so `@rails_commands` will always be nil.

Reply:
@jonathanhefner Nice catch!

I thought about this, but can you think of an elegant way to assert neither of these commands are called?

```diff
--- a/railties/test/generators/app_generator_test.rb
+++ b/railties/test/generators/app_generator_test.rb
@@ -728,10 +728,11 @@ def test_skip_active_job_option
   def test_skip_javascript_option
     generator([destination_root], skip_javascript: true)

-    assert_nil @rails_commands, "`importmap:install` expected to not be called."
-
     run_generator_instance

+    assert_not_includes @rails_commands, "importmap:install", "`importmap:install` expected to not be called."
+    assert_not_includes @rails_commands, "turbo:install stimulus:install", "`turbo:install stimulus:install` expected to not be called."
```